Makefile

```diff
@@ -547,10 +548,9 @@ docker-compose-build:
 	    --cache-from=$(DEV_DOCKER_TAG_BASE)/awx_devel:$(COMPOSE_TAG) .
 
 docker-clean:
-	$(foreach container_id,$(shell docker ps -f name=tools_awx -aq && docker ps -f name=tools_receptor -aq),docker stop $(container_id); docker rm -f $(container_id);)
-	if [ "$(shell docker images | grep awx_devel)" ]; then \
-	  docker images | grep awx_devel | awk '{print $$3}' | xargs docker rmi --force; \
-	fi
+	-$(foreach container_id,$(shell docker ps -f name=tools_awx -aq && docker ps -f name=tools_receptor -aq),docker stop $(container_id); docker rm -f $(container_id);)
+  # TODO: Figure out if it was intentional for the old code to not remove receptor images.
```

Comment:
The receptor images are only used for building images and for hop nodes. This obviously only attempts to deal with the awx_devel image. It also doesn't touch the redis or postgres image, because these are external and should change infrequently. While receptor is more integral and picking up changes are more important, you get this by refreshing the awx_devel image, itself... unless you're building locally or using hop nodes.

I think I agree that in some context it would be appropriate to remove receptor images, but the distinction between `make docker-compose-clean` and `make docker-clean` is, itself, challenging to undertand.